src/ops.rs

```diff
@@ -186,13 +190,20 @@ fn op_now(
   data: libdeno::deno_buf,
 ) -> Box<Op> {
   assert_eq!(data.len(), 0);
-  let start = SystemTime::now();
-  let since_the_epoch = start.duration_since(UNIX_EPOCH).unwrap();
-  let time = since_the_epoch.as_secs() * 1000
-    + u64::from(since_the_epoch.subsec_millis());
+  let start = Instant::now();
+  let since_the_epoch = start.duration_since(_isolate.start_time).as_nanos();
+  let time = since_the_epoch;
```

Comment:
I think this can be simplified as isolate.start_time.elapsed()

I suspect returning (as_secs, subsec_nanos) as the msg is easier / avoids the u64 cast. (This means amending the NowRes table.)

I will give this a whirl soon. Overall this look great.

Reply:
Also need to add libmath crate i think.

Reply:
Yeah, that seems surprising that it's required.

I guess you could just do `time -=(time % REDUCED_TIME_PRECISION)`, would that be right? (e.g. 1000, so 1ms). You'd do that on just the subsec_nanos part.

Reply:
Excellent alternative!

Reply:
Have done some modifications and it's working. Have to rebase everything now.